lib/Client.js

```diff
@@ -21,18 +21,20 @@ const ShardManager = require("./gateway/ShardManager");
 const UnavailableGuild = require("./structures/UnavailableGuild");
 const User = require("./structures/User");
 const VoiceConnectionManager = require("./voice/VoiceConnectionManager");
+const safeRequire = (id, subID = null) => {
```

Comment:
I would also rename the parameter for something more obvious, like `dependency`, `fallbackDependency` or something like that

Reply:
Good idea, done!